content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -526,7 +526,7 @@ please install corresponding CSI drivers.
 * [`gcePersistentDisk`](/docs/concepts/storage/volumes/#gcePersistentDisk) - GCE Persistent Disk
   (**migration on by default** starting v1.23)
 * [`portworxVolume`](/docs/concepts/storage/volumes/#portworxvolume) - Portworx volume
-  (**deprecated** starting v1.25)
+  (**migration on by default** starting v1.31)
```

Comment:
Is it also still deprecated?

Reply:
Sure. It's still deprecated. The intree implementataion will be removed in a future release once the csi migration is completed.